### DISCUSSION TITLE
Label Filter for Task List where no label attached

### DISCUSSION THREAD

**Discussion by shaileshgzp** [madrecha/jamku#1046] [QA] [answered]
### Feature Request

<img width="718" alt="image" src="https://user-images.githubusercontent.com/94695170/167249741-70e73a47-ab60-4ca7-9888-007c6ec8f4a2.png">

Label filter required for Task List where no label. Currently we can filter Task List using Label section with Label assigned to it but if we want to check tasks where no label is assigned then it is not possible. Kindly check and provide solution.

**Comment by adarshmadrecha** [comment, marked as answer]
1. Go to Table View in Task List
2. Sort using "Task Label" Column.
3. This will show all the tasks without any labels.

![image](https://user-images.githubusercontent.com/11911938/167308468-13183f17-d59d-4074-a82e-0076b424f529.png)
